runtime/build.gradle.kts

```diff
@@ -91,6 +113,15 @@ kotlin {
     }
 }
 
+android {
```

Comment:
Note: we unfortunately have to be opinionated about a min and target Android version here...